src/options/index.tsx

```diff
@@ -68,32 +72,48 @@ const OptionsIndex = () => {
               alt="ScombZ Utilities"
               style={{ width: "50%", maxWidth: "250px", margin: "0 auto", display: "block" }}
             />
-            <Button
-              sx={{
-                backgroundColor: "#fff",
-                "&:hover": {
-                  backgroundColor: "#f4f4f4",
-                },
-              }}
-              variant="outlined"
-              onClick={() => {
-                setIsSimple(!isSimple);
-              }}
-            >
-              {isSimple ? "詳細設定へ" : "かんたん設定へ"}
-            </Button>
           </Box>
 
-          {isSimple ? (
-            <SimpleOptions setSettings={setSettings} saves={currentLocalStorage} setSaves={setCurrentLocalStorage} />
-          ) : (
-            <ComplexOptions
-              setSettings={setSettings}
-              saves={currentLocalStorage}
-              setScombzData={setScombzData}
-              setSaves={setCurrentLocalStorage}
-            />
-          )}
+          <Box
+            sx={{
+              borderBottom: 1,
+              borderColor: "divider",
+              position: "sticky",
+              top: 0,
+              left: 0,
+              width: "100%",
+              zIndex: 1,
+              backgroundColor: "#fffa",
+              backdropFilter: "blur(3px)",
+            }}
+          >
+            <Tabs variant="scrollable" scrollButtons="auto" value={currentTab || "basic"}>
+              <Tab component="a" value="basic" href="/options.html?tab=basic" label="基本設定" />
+              <Tab component="a" value="widget" href="/options.html?tab=widget" label="ウィジェット設定" />
+              <Tab component="a" value="advanced" href="/options.html?tab=advanced" label="詳細設定" />
+              <Tab component="a" value="customcss" href="/options.html?tab=customcss" label="カスタムCSS" />
+              <Tab component="a" value="data" href="/options.html?tab=data" label={chrome.i18n.getMessage("IOReset")} />
+              <Tab component="a" value="info" href="/options.html?tab=info" label="情報" />
+            </Tabs>
```

Comment:
Tabをanchorエレメントにするとページ遷移が生じてしまうので、 `history.pushState()` でurlを書き換えつつSPAを保持すると理想のUXを実現できそうです
以下、試してないので動くかわからないですが

```suggestion
            <Tabs variant="scrollable" scrollButtons="auto" value={currentTab || "basic"}
              onChange={(_, value) => {
                setCurrentTab(value);
                history.pushState(null, "", `?tab=${value}`);
              }}
            >
              <Tab value="basic" label="基本設定" />
              <Tab value="widget" label="ウィジェット設定" />
              <Tab value="advanced" label="詳細設定" />
              <Tab value="customcss" label="カスタムCSS" />
              <Tab value="data" label={chrome.i18n.getMessage("IOReset")} />
              <Tab value="info" label="情報" />
            </Tabs>
```

Reply:
あ、↑だとsetCurrentTabがなさそうなのでuseStateにしてください💦

Reply:
いい感じに動きました！ありがとうございます🙇‍♂️